### PR TITLE
Update config.yml

### PR DESCRIPTION
**Pull request by jaydeland**
#### Summary
Setup new site for docs stagging: 
https://docs-stagging.mattermost.com/

#### Ticket Link
https://mattermost.atlassian.net/browse/DOPS-286

### CODE REVIEW

**Comment by justinegeffen** [comment]
Thank you so much, @jaydeland!! :) Do you know offhand whether this has any impact on Sphinx builds? I'll take a look through the config files etc today and make sure.

**Comment by justinegeffen** [comment]
Oh, and I noted that staging is a protected branch which is awesome. Is it based on GitHub group permissions?

**Review by justinegeffen — COMMENTED** [review]
This makes a lot of sense - I've just provided feedback on the naming (hopefully it's easy to change!) :)

File: .circleci/config.yml
Comment (cwarnermm) [inline .circleci/config.yml:48]:
```suggestion
          to: 's3://docs-staging.mattermost.com/'
```

**Review by cwarnermm — COMMENTED** [review]
Minor suggestions left in-line to correct spelling of "staging"

**Comment by jaydeland** [comment]
The DNS entry will need to be updated to match by @metanerd

**Review by cwarnermm — APPROVED** [review]
Looks great! Thank you!

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
@metanerd, @cwarnermm: what's the status on this? Can we merge it?

**Comment by cwarnermm** [comment]
@metanerd @justinegeffen - no further changes from my side!

**Comment by justinegeffen** [comment]
The only change I'd make is the actual branch name which is still "stagging". But other than that I'm happy. :)

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
I can't seem to update the branch though: "Update branch attempt failed Couldn't update "stagging": At least 1 approving review is required by reviewers with write access."

**Comment by justinegeffen** [comment]
I can indeed! Thanks @jaydeland! :)

**Comment by justinegeffen** [comment]
Oh, one last thing: Is it possible to rename the branch to "staging" before merge? Or is it best to do it post-merge?

**Comment by jaydeland** [comment]
@JustinReynolds-MM - Indeed it looks like the config is already trying to key off 'staging' and will therefore not trigger jobs. I tried to change it, but no longer have permission to push changes to this repo.

**Comment by justinegeffen** [comment]
I'm happy to change it - what do I need to do?

**Comment by jaydeland** [comment]
@justinegeffen you will need to sync the repo, checkout the stagging branch, rename it and then push that change.
git checkout stagging
git branch -m stagging staging
git push

**Pull request merged**
